recipes/servicestack/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "servicestack" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4874987b82e81a8eeb146b53df6b282f48f843aa1e4f17eecc24d723dc3c5247
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - dataclasses >=0.6
+    - dataclasses-json >=0.5.4
+    - marshmallow >=3.12.2
+    - python >=3.9
+    - requests >=2.25.1
+
+test:
+  imports:
+    - servicestack
+    - tests
```

Comment:
Is importing (not running) `tests` (which has an empty `__init__.py`) really useful for validating that the package isn't broken?

Reply:
Still learning ins and outs of conda and the environments `conda build` is constructing, so this was a mistake on my part.

Reply:
I tried wiring these tests to work in the conda environment by using `test` `commands:`, do you have any suggestions how to select the correct `python` version (in this case 3.9)?. Using `${PREFIX}/bin/python` seems to have 2.7 and `${PREFIX}/bin/python3` is 3.8 enough though the meta.yaml specifies only >=3.9.

I can't find a clear way of doing this in the docs, but likely I'm missing something.

Reply:
I've run with `python3` for test commands for now as I want to keep the conda specifics isolated to the meta.yaml so that we don't have to have full pypi package releases including test environment adjustments just to get through the conda-forge release cycle. Any tips to achieve this in a better way I'd greatly appreciate. Thanks for the review!